survey.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   

```

Comment:
I suggest you remove the unnecessary blank line. Adding blank lines without reason is not a good practice in HTML coding. Check out [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for HTML & CSS best practices.